spring-cloud-starter-aws-secrets-manager-config/src/test/java/io/awspring/cloud/autoconfigure/secretsmanager/AwsSecretsManagerConfigDataLocationResolverTest.java

```diff
@@ -53,6 +53,17 @@ void testResolveProfileSpecificWithCustomPaths() {
 		assertThat(toContexts(locations)).containsExactly("/mypath1", "/mypath2", "/mypath3");
 	}
 
+	@Test
+	void testResolveProfileSpecificWithCustomPathsOptional() {
+		String location = "aws-secretsmanager:optional /mypath1;/mypath2;optional /mypath3";
```

Comment:
Why a space? Isn't it idiomatic to support: `aws-secretsmanager:optional:/mypath1;/mypath2;optional:/mypath3`?

Reply:
Agree it is better to use `:`